java/urllink/src/main/resources/dev/application.properties

```diff
@@ -0,0 +1,66 @@
+#data source pool
+spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
+spring.datasource.url=jdbc:mysql://10.0.0.95:3306/urllink?useAffectedRows=true&useUnicode=true&characterEncoding=utf-8&useSSL=false&serverTimezone=GMT%2B8&zeroDateTimeBehavior=convertToNull
+spring.datasource.username=root
+spring.datasource.password=password
+spring.datasource.initialSize=10
+spring.datasource.maxActive=100
+spring.datasource.maxIdle=20
+spring.datasource.minIdle=10
+spring.datasource.maxWait=2000
+spring.datasource.removeAbandonedTimeout=180
+spring.datasource.removeAbandoned=true
+spring.datasource.time-between-eviction-runs-millis=30000
+spring.datasource.test-on-borrow=false
+spring.datasource.test-while-idle=true
+spring.datasource.jdbcInterceptors=org.apache.tomcat.jdbc.pool.interceptor.SlowQueryReport(logFailed=true)
+
+#redis
+spring.redis.database=0
+spring.redis.host=47.97.168.130
+spring.redis.port=6379
+spring.redis.password=dfeaf685ab5aa9b05dcdecbc51cab9b2

```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- e45ac27c-24f9-4047-b693-1b4c11237506 -->